org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/quickfix/DiagnosticResolution.java

```diff
@@ -131,7 +134,8 @@ public WorkspaceEdit apply() {
 			}
 			return edit;
 		} catch (Exception exc) {
-			throw new WrappedException(exc);
+			log.error("Creation of WorkspaceEdit failed.", exc);
```

Comment:
@heinrichWeichert are you ok with this change?

Reply:
Could you we return a null-Edit instead?

Reply:
Yes, we could return `new WorkspaceEdit()`, LSP defines the workspaceEdit as a nullable field. I checked LSP4E and it always checks for the field to be != null before accessing it, I also checked what it would do with an empty edit, and it will have the same result, as it will contain no changes. I think returning null is simpler, but if you think returning a `new WorkspaceEdit()` is better, I would do that, I do not have a strong opinion. May I ask why you think that returning an empty Edit is preferable?

Reply:
@szarnekow  what do you think?

Reply:
@rubenporras OK, i didn't check that. If it is indeed nullsafe an expected it's fine with me.

Reply:
I've a preference towards returning null, too. An empty edit suggests that there is something the client *could* apply which might mess with its undo queue or whatsoever.

Reply:
i am not sure if and how tracing works in standalone mode.

Reply:
all fine for me